common/common-rest/src/main/java/org/apache/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -160,16 +163,24 @@ public void invoke() {
   }
 
   protected Response prepareInvoke() throws Throwable {
-    this.initProduceProcessor();
+    SCBStatus currentStatus = SCBEngine.getInstance().getStatus();
+    if (SCBStatus.UP.equals(currentStatus)) {
+      this.initProduceProcessor();
 
-    this.setContext();
-    invocation.getHandlerContext().put(RestConst.REST_REQUEST, requestEx);
+      this.setContext();
+      invocation.getHandlerContext().put(RestConst.REST_REQUEST, requestEx);
 
-    for (HttpServerFilter filter : httpServerFilters) {
-      Response response = filter.afterReceiveRequest(invocation, requestEx);
-      if (response != null) {
-        return response;
+      for (HttpServerFilter filter : httpServerFilters) {
+        Response response = filter.afterReceiveRequest(invocation, requestEx);
+        if (response != null) {
+          return response;
+        }
       }
+    } else {
+      LOGGER.warn(
+          "System is starting and not ready for remote calls or shutting down in progress, STATUS = " + currentStatus);
```

Comment:
The Request is rejected, as the service provider cannot  process the request due to STATUS =  (you can add the status information here).

Reply:
OK,it seems better,Done